.github/workflows/install.yaml

```diff
@@ -33,12 +33,6 @@ jobs:
         run: |
           ./utils/installer/install.sh --local --no-install-dependencies
 
-      - name: Test LunarVim PackerCompile
-        run: if "$HOME"/.local/bin/lvim --headless +PackerCompile -c ':qall' 2>&1|grep -q 'Error'; then false; fi
-
-      - name: Test LunarVim Health
-        run: if "$HOME"/.local/bin/lvim --headless +checkhealth -c ':qall' 2>&1|grep -q 'Error'; then false; fi
```

Comment:
is there a reason we are removing health check and packercompile test from our CI?